test/integration/pdf_generation_test.exs

```diff
@@ -394,6 +394,25 @@ defmodule ChromicPDF.PDFGenerationTest do
     end
   end
 
+  describe "timezone can be set thru print_to_pdf/2" do
+    setup do
+      start_supervised!({ChromicPDF, offline: false})
```

Comment:
Is the `offline: false` necessary for this test?

Reply:
Sorry about that, @maltoe - it is not required. I've removed it!